docs/ensnode.io/src/styles/starlight.css

```
@@ -200,6 +200,19 @@ a[rel="prev"]:hover {
   color: var(--sl-color-accent);
 }
 
+.sl-markdown-content > p a:not(:has(code)),
+.sl-markdown-content :is(ul, ol) > li a:not(:has(code)),
+.starlight-aside__content a {
+  text-decoration: underline;
+  text-underline-offset: 4px;
+
+  transition: all 0.2s ease-in-out;
```

Stylelint (17.11.1)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)